int32 min
int32 max
---
int32 PosX
int32 PosY
